etc/spack/defaults/packages.yaml

```diff
@@ -39,3 +39,4 @@ packages:
       scalapack: [netlib-scalapack]
       szip: [libszip, libaec]
       tbb: [intel-tbb]
+      unwind: [libunwind]
```

Comment:
This might be the most minor change request I've ever put in on a PR, but it would be nice to have a newline here.

Reply:
Added in [3099599](https://github.com/spack/spack/pull/8825/commits/3099599033a524cabfa56bed405a4a0fae072b86).